### DISCUSSION TITLE
Add invited user information such as first name and last name

### DISCUSSION THREAD

**Discussion by Mbengparis** [directus/directus#5583] [Archive]
How can i permit a user to enter their information(names) when registering a new user using the  user invite and accept link endpoint. 

To accept invite, a POST request is made to /users/invite/accept endpoint with body { token, password } and the response is an empty body.

Is there a way to customize the accept invite endpoint to be able to send also the user data so that the body will now be:

```json
{
  "token": token ,
  "password": password,
  "first_name": firstname,
  "last_name": lastname
}
```

If yes where can i customize it?. I have searched in docs but to no avail.
 
Thanks in advance

**Comment by Mbengparis** [comment]
@rijkvanzanten i need help please

**Comment by rijkvanzanten** [comment]
This is not currently possible. You can use the [update-a-user](https://docs.directus.io/reference/api/system/users/#update-a-user) method to update the user after you've created it through the invite

**Reply by Mbengparis** [reply]
ok. thanks

**Comment by rhclayto** [comment]
Hi,

Is there any desire to implement this on the development team's part? I was looking for this functionality too. The 'update-a-user' link in the response above is currently a 404.

**Reply by rijkvanzanten** [reply]
Updated update a user link: https://docs.directus.io/reference/system/users.html#update-a-user
